src/main/java/Duke.java

```diff
@@ -1,10 +1,375 @@
+//Matric Number: A0200273X
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+
+
+    private final static String Dir = ".\\task.txt";
+    private static ArrayList<String> tasks = new ArrayList<>();
+
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+        writeFile(Dir,"\n");
+
+        System.out.print("Tasks saved previously: \n");
+        printLines();
+        retrieveFile(Dir);
+        showWelcomeMessage();
+        //String Array of size 100 created to store item list
+        String[] itemList = new String[100];
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+
+        Duke numTasks = new Duke();
+        int taskCounts = 0;
+
+        /**
+         * When user enter list it will show a list of input saved
+         * to do command add a item into the list and mark it as task
+         * deadline command set a item with the deadline and mark it as deadline
+         * Event command add item as events to do and mark it as event
+         * When user enter done it will mark the task as done into the list
+         * It will run continuously till bye command is given
+         */
+        while (!line.equals("bye")) {
+            for (int i = 0; i < 100; i++) {
```

Comment:
I'm not sure why loop is needed here. Maybe can comment to clarify this line, or use a variable name that explains better?

Reply:
If this loop is to record the current index of `itemList`, maybe you can find an alternative to not use loop and at the same time avoid deep nesting?